3001-4000/LIT3955Homily.xml

```diff
@@ -6,7 +6,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          <titleStmt>
             <title xml:lang="gez" xml:id="t1">ድርሳን፡ ዘደረሰ፡ አብ፡ ክቡር፡ ወቅዱስ፡ መምህር፡ አርኬ፡ ላኦስ፡ ኤጲስ፡ ቆጶስ፡ ዘሀገረ፡ ደና፡ በእንተ፡ ገብርኤል፡ ሊቀ፡ መላእክት።</title>
             <title xml:lang="gez" corresp="#t1" type="normalized">Dǝrsān za-darasa ʾab kǝbur wa-qǝddus mamhǝr ʾArke Lāʾos ʾeṗṗis qoṗos za-hagara Danā baʾǝnta Gabrǝʾel liqa malāʾǝkt</title>
-            <title xml:lang="en" corresp="#t1">Homily of Archelaus of Dana on the Archangel Gabriel</title>
```

Comment:
What do you think about adding for the sake of future catalogers a note that the attribution to Archelaus is not always present? It is missing e. g. in BNFabb29 or BerOrFol117?